### ISSUE TITLE
How to look up the tokens based on the word representation?

### ISSUE THREAD

**Issue by PhyllisH** [allenai/bilm-tf#62]
Hi~
I'm trying to use the ELMO replacing the original embedding lookup word representation in my model. And  I see that you have a model in tensorflow https://tfhub.dev/google/elmo/2. But based on the example, I can only generate word representation and there is no api for the representation back to words. Could u help me with that? 
Thanks

**Comment by matt-peters** [comment]
I'm not sure what you mean by an API to generate the representation back to words?  To generate text from the LM?

**Comment by PhyllisH** [comment]
Yes. Can I use the token matrix as embedding matrix？



------------------ Original ------------------

**Comment by matt-peters** [comment]
We have a separate softmax matrix for prediction.  You can get it as part of the tensorflow checkpoint, see these comments I just added to the README: https://github.com/allenai/bilm-tf/blob/master/README.md#are-the-softmax-weights-available
